src/schema/v2/partner/BulkOperation/bulkUpdateArtworksMetadataMutation.ts

```diff
@@ -94,6 +95,10 @@ const BulkUpdateArtworksMetadataInput = new GraphQLInputObjectType({
       type: GraphQLBoolean,
       description: "Publish or unpublish artworks",
     },
+    signatureDetails: {
```

Comment:
CMS batch edit has a nice `dev` mode interface

It also shows us how the client is sending over this field to MP, looks a like `signature`!

<img width="2047" height="810" alt="Screenshot 2025-07-21 at 2 39 08 PM" src="https://github.com/user-attachments/assets/8866a9bc-2950-42fe-b0a7-dd3dd5010a37" />

Reply:
let me know if you want to test together!